atom/common/api/atom_bindings.cc

```diff
@@ -7,20 +7,26 @@
 #include <algorithm>
 #include <iostream>
 #include <string>
+#include <vector>
 
 #include "atom/common/api/locker.h"
 #include "atom/common/atom_version.h"
 #include "atom/common/chrome_version.h"
 #include "atom/common/heap_snapshot.h"
 #include "atom/common/native_mate_converters/file_path_converter.h"
 #include "atom/common/native_mate_converters/string16_converter.h"
-#include "atom/common/node_includes.h"
+#include "atom/common/promise_util.h"
 #include "base/logging.h"
+#include "base/process/process_handle.h"
 #include "base/process/process_info.h"
 #include "base/process/process_metrics_iocounters.h"
 #include "base/sys_info.h"
 #include "base/threading/thread_restrictions.h"
 #include "native_mate/dictionary.h"
+#include "services/resource_coordinator/public/cpp/memory_instrumentation/global_memory_dump.h"
+#include "services/resource_coordinator/public/cpp/memory_instrumentation/memory_instrumentation.h"
+// Must be the last in the includes list.
+#include "atom/common/node_includes.h"
```

Comment:
Please make sure that `clang-format` won't "fix" the order of includes here.
Usually leaving an empty line before such an include helps.

Reply:
I thought the comment helps, and prevents sorting. But will verify, thanks.